src/components/notifications/Webpush.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <button
+        class="mb-3"
+        type="button"
+        :class="[
+            'btn',
+            canRegister ? 'btn-primary' : 'btn-danger'
+        ]"
+        :disabled="!btnEnabled"
```

Comment:
this does not look right
Am I missing something?
`btnEnabled` is initalised as false => this is always default-disabled?

Reply:
Thats correct, this will intialize as disabled. Not all browsers are able to do push nothification, we will only enable the button after we check that the browser supports this functionality.